src/Entity/Hospital.java

```diff
@@ -81,4 +87,11 @@ public void addOffices(String s, Office off) {
         return buildings;
     }
 
+    public ObservableList<Kiosk> getKiosks () { return kiosks; }
+    public Kiosk getCurrentKiosk() { return  currentKiosk; }
+
+    public void addKiosk (Kiosk newKiosk) {
+        kiosks.add(newKiosk);
```

Comment:
Okay so you already have this!  Just change visibility of the variable and then switch over the boundary code to use these functions